tests/baselines/reference/classPropertyErrorOnNameOnly.errors.txt

```diff
@@ -1,8 +1,8 @@
 tests/cases/compiler/classPropertyErrorOnNameOnly.ts(7,3): error TS2322: Type '(val: Values) => "1" | "2" | "3" | "4" | "5" | undefined' is not assignable to type 'FuncType'.
-  Type '"1" | "2" | "3" | "4" | "5" | undefined' is not assignable to type 'string'.
+  Type 'string | undefined' is not assignable to type 'string'.

```

Comment:
I... _think_ this change aligns with the requested change in the issue? But I can take a deeper look if requested.